madgraph/iolibs/template_files/madevent_driver.f

```diff
@@ -71,6 +71,7 @@ Program DRIVER
 c      integer ncols,ncolflow(maxamps),ncolalt(maxamps),ic
 c      common/to_colstats/ncols,ncolflow,ncolalt,ic
 
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc